### ISSUE TITLE
Reopen: Legal Definition Conflict in License

### ISSUE THREAD

**Issue by LokiFawkes** [WinampDesktop/winamp#251]
The License is clearly proprietary, forbidding distribution and public modification, only allowing users to read the source code and to privately run their modified versions. However, it describes itself as copyleft.

This is a potential conflict in legal definition, as the definition of copyleft as accepted by society at large forbids this type of license. This type of conflict has undefined behavior in copyright law. A court could, for example, rule that your license is misleading and unenforceable, ranging from a need to edit the License once again to make it enforceable, to invalidating the License entirely and releasing the code to the hands of the public.

In addition, if anyone forks Winamp as part of the process of creating and testing modifications to be staged for a pull request, they would run afoul of the license for the same reason as above - It forbids distribution, and forking on Github is distribution, but also a necessary part of contribution.

Suggested fix: Change to a copyleft license as promised by the preamble of the License, or move Winamp's source code to a more restrictive platform, such as an internally controlled gitlab server.

Reopening because the repo is still violating TOS.

**Comment by buseche** [comment]
Guys, one suggestion:

Please, pleaseeeee, request a IP Lawer, consult this mess, and wait his answer about this matter. Obiously Nullsoft before release the program, i choose believe they asked to his lawers about licensing.

Many software is private with an community edition, i don't understand why you open 12371239486239 Issues about the license.

Winamp is an important part of early 2000 internet, and we can watch how the magic goes.

**Comment by LokiFawkes** [comment]
This isn't really a community edition, with the license they're using. It's just straight up proprietary. Also as another user mentioned, Winamp may contain **stolen** FOSS code.

**Comment by buseche** [comment]
Again, this code **BELONGS** Nullsoft, they can with his code anything, and they choose upload to Github, again, consult to an IP Lawyer, or ask to WIPO (https://www.wipo.int/), i don't know why you do you want to redistribute this app when exists **A LOT** of players with the same functions.

If you have evidences about a code store, share with the authorities.

**Comment by LokiFawkes** [comment]
1) Copyleft has a well known legal definition, and the license is not copyleft, but claims to be.
2) Github TOS D5
3) Stolen code discoveries are being made constantly and added to Issues. Maybe try keeping up.

**Comment by LokiFawkes** [comment]
An example of stolen code confirmed:
From #265 courtesy of KingDuckZ who contacted DiffieHellman on liberachat:

> Yep, winamp confirmed to have been infringing the GPLv2, LGPLv2 and LGPLv2.1 all this time
> I would argue that everyone is free to delete the current proprietary license and go with the GPLv2
> The licensing is really a compete mess and it'll take serious analysis to work out which licenses apply, but what I can see, the current proprietary license is not compatible with either the; LGPLv2, LGPLv2.1, GPLv2 or MPLv1.1
> It'll take a lot of work to track down the license headers, as I'm seeing a lot of deleted license headers - which is a serious copyright violation.
> Src/Plugins/Visualization/vis_avs/evallib/bison shows traces of GNU bison

And this isn't even getting into the Dolby code being unearthed.

**Comment by dontfwiththestream** [comment]
Thanks for the heads up, I have relicensed my fork to GPLv2

**Comment by LokiFawkes** [comment]
> Thanks for the heads up, I have relicensed my fork to GPLv2

Good luck finding and replacing the dolby stuff

**Comment by morsik** [comment]
> Again, this code **BELONGS** Nullsoft, they can with his code anything, and they choose upload to Github

@buseche dude, wake up... were you in coma or what? Nullsoft doesn't exists already for 10 years. Some shady company bought Winamp source code, assets, copyrights and other stuff at the end of Nullsoft's life from them.

**Comment by KingDuckZ** [comment]
> Thanks for the heads up, I have relicensed my fork to GPLv2

@dontfwiththestream What happened to your fork? It's gone now

**Comment by KingDuckZ** [comment]
Right, and after 10 years of stealing other people's work now they'll just quietly accept it. I'd love to get proven wrong but I wouldn't keep my hopes up, what with the whole initial "free labour from the community we stole code from" licence and the poor attitude so far
